extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -207,6 +207,7 @@ export class Bind {
   /**
    * @param {!Window} embedWin
    * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
+   * @nocollapse
```

Comment:
Thanks for figuring this out.

Reply:
I just verified that this PR reinstates all 8 instances of `@nocollapse` that were inadvertently removed by #22446 (see https://github.com/ampproject/amphtml/pull/22446/commits/206027c763bfb227eea98d9b5ae630f7202777e5)

/cc @lannka